compiler/src/dotty/tools/dotc/transform/patmat/Space.scala

```diff
@@ -98,19 +102,31 @@ trait SpaceLogic {
   def show(sp: Space): String
 
   /** Simplify space using the laws, there's no nested union after simplify */
-  def simplify(space: Space): Space = space match {
+  def simplify(space: Space, aggressive: Boolean = false): Space = space match {
```

Comment:
Could you update the documentation of this method to explain what the `aggressive` parameter is for?